src/rules/importBlacklistRule.ts

```diff
@@ -41,7 +41,7 @@ export class Rule extends Lint.Rules.AbstractRule {
             In other cases, you may simply want to ban an import because using
             it is undesirable or unsafe.`,
         optionsDescription:
-            "A list of blacklisted modules, named imports, or regular expression patterns.",
+            "A list of blacklisted modules, named imports, or an array of expression patterns.",
```

Comment:
I think we should keep the word "regular" in there; it should say "... regular expression ..."

Reply:
Done